### PR TITLE
i18n(es): update multiple pages

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- Update `/core-concepts/framework-components.md` to match changes in #862 
- Update `/editor-setup.md` to match changes in #831
- Update `/guides/aliases.md` to match changes in #871 
- Update `/guides/configuring-astro` to match changes in #931
- Update `/guides/typescript` to match changes in #956

### CODE REVIEW

File: src/pages/es/editor-setup.md [hunk 1]
```diff
@@ -32,8 +32,44 @@ Nuestra increíble comunidad mantiene extensiones para otros editores de código
...
+Consulte el [README del plugin de Prettier](https://github.com/withastro/prettier-plugin-astro/blob/main/README.md) para obtener más información sobre sus opciones, cómo configurar Prettier dentro de VS Code y más.
+
+:::caution[Uso con pnpm]
+Debido a problemas dentro de Prettier, el complemento no se detectará automáticamente al usar [pnpm](https://pnpm.io/). Para que encuentre el plugin, se debe agregar el siguiente parámetro al ejecutar Prettier:
+
+```shell
+prettier --write --plugin-search-dir=. .
+```
+
+También se requieren configuraciones adicionales cuando se usa Prettier dentro de VS Code. Consulte el README del plugin para obtener más información.
```
Comment (agustinmulet) [inline src/pages/es/editor-setup.md:74]:
```suggestion
También se requieren configuraciones adicionales cuando se usa Prettier dentro de VS Code. Consulta el README del plugin para obtener más información.
```

File: src/pages/es/guides/configuring-astro.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Configuración de Astro
 i18nReady: true
 ---
 
-Personalice cómo funciona Astro agregando un archivo `astro.config.mjs` en su proyecto. Este es un archivo común en todos los proyectos de Astro; todas plantillas y temas de ejemplos oficiales se encuentran con uno de forma predeterminada.
+Personalice cómo funciona Astro agregando un archivo `astro.config.mjs` en tu proyecto. Este es un archivo común en todos los proyectos de Astro; todos los ejemplos oficiales como plantillas y temas cuentan con uno de forma predeterminada.
```
Comment (agustinmulet) [inline src/pages/es/guides/configuring-astro.md:7]:
```suggestion
Personalice cómo funciona Astro agregando un archivo `astro.config.mjs` en tu proyecto. Este es un archivo común en todos los proyectos de Astro; todos los ejemplos oficiales, sean plantillas o temas, cuentan con uno de forma predeterminada.
```

File: src/pages/es/guides/configuring-astro.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: Configuración de Astro
...
 
 📚 Lea la [referencia de configuración](/es/reference/configuration-reference/) de Astro para obtener una descripción general y completa de todas las opciones de configuración.
```
Comment (agustinmulet) [inline src/pages/es/guides/configuring-astro.md:9]:
```suggestion
📚 Lee la [referencia de configuración](/es/reference/configuration-reference/) de Astro para obtener una descripción general y completa de todas las opciones de configuración.
```

File: src/pages/es/guides/configuring-astro.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ astro build --config my-config-file.js
 
 ## Configurar Intellisense
 
-Astro recomienda usar `defineConfig()` en su archivo de configuración. `defineConfig()` proporciona IntelliSense automático para su IDE. Los editores como VSCode pueden leer las definiciones de tipo TypeScript de Astro y proporcionar sugerencias de tipo jsdoc automáticas, incluso si su archivo de configuración no está escrito en TypeScript.
+Astro recomienda usar `defineConfig()` en tu archivo de configuración. `defineConfig()` proporciona IntelliSense automático para su IDE. Los editores como VSCode pueden leer las definiciones de tipo TypeScript de Astro y proporcionar sugerencias de tipo jsdoc automáticas, incluso si su archivo de configuración no está escrito en TypeScript.
```
Comment (agustinmulet) [inline src/pages/es/guides/configuring-astro.md:56]:
```suggestion
Astro recomienda usar `defineConfig()` en tu archivo de configuración. `defineConfig()` proporciona IntelliSense automático para tu IDE. Los editores como VSCode pueden leer las definiciones de tipo TypeScript de Astro y proporcionar sugerencias de tipo jsdoc automáticas, incluso si tu archivo de configuración no está escrito en TypeScript.
```

File: src/pages/es/guides/typescript.md [hunk 5]
```diff
@@ -5,15 +5,15 @@ description: Aprenda a usar TypeScript incorporado en Astro.
...
+Astro no realiza ninguna verificación de tipos por sí mismo. La verificación de tipos debe realizarse fuera de Astro, ya sea por el IDE o mediante un script separado. La [extensión de Astro VSCode](/es/editor-setup/) proporciona automáticamente sugerencias y errores de TypeScript en tus archivos abiertos.
 
 ## Configuración
 
-Se **recomienda encarecidamente** que cree un archivo `tsconfig.json` en su proyecto, para que las herramientas como Astro y VSCode sepan interpretarlo. Algunas funciones (como las importaciones de paquetes npm) no son totalmente compatibles con TypeScript a menos que crees un archivo `tsconfig.json`.
+Se **recomienda encarecidamente** que cree un archivo `tsconfig.json` en su proyecto, para que las herramientas como Astro y VSCode sepan interpretar tu proyecto. Algunas funcionalidades (como las importaciones de paquetes npm) no son totalmente compatibles con TypeScript a menos que crees un archivo `tsconfig.json`.
```
Comment (agustinmulet) [inline src/pages/es/guides/typescript.md:14]:
```suggestion
Se **recomienda encarecidamente** que crees un archivo `tsconfig.json` para que las herramientas como Astro y VSCode sepan cómo interpretar tu proyecto. Algunas funcionalidades (como las importaciones de paquetes npm) no son totalmente compatibles con TypeScript a menos que crees un archivo `tsconfig.json`.
```

File: src/pages/es/guides/typescript.md [hunk 6]
```diff
@@ -5,15 +5,15 @@ description: Aprenda a usar TypeScript incorporado en Astro.
...
 
-Algunas opciones de configuración de TypeScript requieren atención especial en Astro. A continuación le recomendamos un archivo `tsconfig.json` básico, que puede copiar y pegar en su proyecto. Cada [plantilla en astro.new](https://astro.new/) incluye este archivo `tsconfig.json` por defecto.
+Algunas opciones de configuración de TypeScript requieren atención especial en Astro. A continuación le recomendamos un archivo `tsconfig.json` básico, que puedes copiar y pegar en tu proyecto. Cada [plantilla en astro.new](https://astro.new/) incluye este archivo `tsconfig.json` por defecto.
```
Comment (agustinmulet) [inline src/pages/es/guides/typescript.md:16]:
```suggestion
Algunas opciones de configuración de TypeScript requieren atención especial en Astro. A continuación te recomendamos un archivo `tsconfig.json` básico, que puedes copiar y pegar en tu proyecto. Cada [plantilla en astro.new](https://astro.new/) incluye este archivo `tsconfig.json` por defecto.
```

File: src/pages/es/guides/typescript.md [hunk 7]
```diff
@@ -66,7 +68,7 @@ import Layout from '@layouts/Layout.astro';
 
 ## Props de componentes
 
-Astro soporta escribir las props de su componente de Astro en TypeScript. Para habilitarlo, exporte una interfaz TypeScript `Props` desde su componente de Astro. La [extensión de Astro VSCode](/es/editor-setup/) buscará automáticamente la exportación de `Props` y le brindará el autocompletado adecuado de TS cuando use ese componente dentro de otra plantilla.
+Astro soporta escribir las props de los componentes de Astro en TypeScript. Para habilitarlo, exporte una interfaz TypeScript `Props` desde tu componente de Astro. La [extensión de Astro VSCode](/es/editor-setup/) buscará automáticamente la exportación de `Props` y le brindará el autocompletado adecuado de TS cuando use ese componente dentro de otra plantilla.
```
Comment (agustinmulet) [inline src/pages/es/guides/typescript.md:71]:
```suggestion
Astro soporta escribir las props de los componentes de Astro en TypeScript. Para habilitarlo, exporta una interfaz TypeScript `Props` desde tu componente de Astro. La [extensión de Astro VSCode](/es/editor-setup/) buscará automáticamente la exportación de `Props` y te brindará el autocompletado adecuado de TS cuando uses ese componente dentro de otra plantilla.
```

File: src/pages/es/guides/typescript.md [hunk 8]
```diff
@@ -80,6 +82,19 @@ const { greeting = 'Hello', name } = Astro.props
 <h2>{greeting}, {name}!</h2>
 ```
 
+## Verificación de tipos
+
+Para ver errores de tipos en tu editor, asegúrese de tener instalada la [extensión de Astro VS Code](/es/editor-setup/). Tenga en cuenta que los comandos `astro start` y `astro build` transpilarán el código con esbuild, pero no ejecutarán ninguna verificación de tipos. Para evitar que su código se compile si contiene errores de TypeScript, cambie su script de "compilación" en `package.json` a lo siguiente:
```
Comment (agustinmulet) [inline src/pages/es/guides/typescript.md:87]:
```suggestion
Para ver errores de tipos en tu editor, asegúrate de tener instalada la [extensión de Astro VS Code](/es/editor-setup/). Ten en cuenta que los comandos `astro start` y `astro build` transpilarán el código con esbuild, pero no ejecutarán ninguna verificación de tipos. Para evitar que tu código se compile si contiene errores de TypeScript, cambia tu script de "compilación" en `package.json` a lo siguiente:
```

File: src/pages/es/guides/typescript.md [hunk 9]
```diff
@@ -80,6 +82,19 @@ const { greeting = 'Hello', name } = Astro.props
...
+```diff
+-    "build": "astro build",
++    "build": "astro check && tsc --noEmit && astro build",
+```
+
+:::note
+`astro check` solo verifica los tipos dentro de los archivos `.astro`, y `tsc --noEmit` solo verifica los tipos dentro de los archivos `.ts` y `.tsx`.
+:::
+
 📚 Lea más sobre las [importaciones de archivos `.ts`](/es/guides/imports/#typescript) en Astro.
```
Comment (agustinmulet) [inline src/pages/es/guides/typescript.md:98]:
```suggestion
📚 Lee más sobre las [importaciones de archivos `.ts`](/es/guides/imports/#typescript) en Astro.
```

File: src/pages/es/guides/typescript.md [hunk 10]
```diff
@@ -80,6 +82,19 @@ const { greeting = 'Hello', name } = Astro.props
...
 
 📚 Lea más sobre la [configuración de TypeScript](https://www.typescriptlang.org/tsconfig/).
```
Comment (agustinmulet) [inline src/pages/es/guides/typescript.md:100]:
```suggestion
📚 Lee más sobre la [configuración de TypeScript](https://www.typescriptlang.org/tsconfig/).
```

**Review by agustinmulet — COMMENTED** [review]
Simple suggestions, keep up the good work! 😤

**Comment by kevinzunigacuellar** [comment]
LGTM!

**Comment by Yan-Thomas** [comment]
Triple merge 😎. Amazing work! 🚀

**Pull request merged**